main.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func main() {
-	if len(os.Args) < 1 {
+	if len(os.Args) <= 1 {
```

Comment:
seria melhor colocar <2 por ser mais facil de ler
```suggestion
	if len(os.Args) <2  {
```

Reply:
Gostei